src/javascript/core/send.js

```diff
@@ -29,29 +29,51 @@ function createXMLHttp() {
 		// "withCredentials" only exists on XMLHTTPRequest2 objects.
 		if (!utils.isUndefined(xmlHttp.withCredentials)) {
 			return {
-				xmlHttp: xmlHttp,
-				XDomainRequest: false
+				send: function (domain, path) {
+					xmlHttp.open('POST', domain, true);
+					xmlHttp.setRequestHeader("Content-type", "application/json");
+					xmlHttp.send(path);
+				},
+				complete: function (callback) {
+					xmlHttp.onerror = function () {
+						callback(this, xmlHttp);
+					};
+					xmlHttp.onload = function () {
+						if (xmlHttp.status >= 200 && xmlHttp.status < 300) {
+							callback(null, xmlHttp);
+						} else {
+							callback('Incorrect response: ' + xmlHttp.status, xmlHttp);
+						}
+					};
+				}
 			};
 		}
-
-		// Otherwise, check if XDomainRequest.
-		// XDomainRequest only exists in IE, and is IE's way of making CORS requests.
-		if (!utils.isUndefined(window.XDomainRequest)) {
-			return {
-				xmlHttp: new window.XDomainRequest(),
-				XDomainRequest: true
-			};
+	} catch (error) {}
```

Comment:
It doesn't matter if there is an error, it will fall back to the image tag method just below.